modules/administration-guide/partials/assembly_migration-from-postgresql-9-to-postgresql-13.adoc

```diff
@@ -0,0 +1,30 @@
+
+
+:parent-context-of-migration-from-postgresql-9-to-postgresql-13: {context}
+
+[id="migration-from-postgresql-9-to-postgresql-13_{context}"]
+= Migration from PostgreSQL 9 to PostgreSQL 13
+
+:context: migration-from-postgresql-9-to-postgresql-13
+
+By the 11th of November, 2021, the PostgreSQL version 9.6 came out of support, and {prod-short} team recommends that all users undergo migrating to version 13.
```

Comment:
This looks like a release note to me. If we need to tell this information to customers, I propose that we create a release note and link to this procedure from the release note.
Let me know what you think about this possibility.